prebuilt_swift_static_framework/prebuilt_swift_static_framework.bzl

```diff
@@ -76,8 +107,8 @@ _prebuilt_swift_static_framework = rule(
             providers = [CcInfo, SwiftInfo],
             cfg = apple_common.multi_arch_split,
         ),
-        framework_name = attr.string(mandatory = True),
-        minimum_os_version = attr.string(default = "10.0"),
+        module_name = attr.string(mandatory = True),
+        minimum_os_version = attr.string(default = _DEFAULT_MINIMUM_OS_VERSION),
```

Comment:
Is the default used when a value of `None` has been given? With these changes, the macro will now pass `None` for this attr if the macro itself is not called with a `minimum_os_version` value.